### DISCUSSION TITLE
2.6.9 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#2759] [讨论]
· 高级启动设置优化
　　为版本独立设置添加“忽略 Java 兼容性警告”选项（#2361，@SALTWOOD）
　　若忽略 Java 兼容性警告，则不再显示相关提示
　　版本独立设置中的“文件校验”改为复选框
· 当启动器更新被拦截时显示相关帮助（#2583，@mcwuguodadi）
· 安装 CurseForge 整合包时支持选择是否安装可选 Mod（#2639，@hejiehao）
· 安装 MultiMC 整合包时支持迁移 MultiMC 的部分设置（#2655，@zkitefly）
　　包括：版本图标、启动前执行命令、自动进入服务器、忽略 Java 兼容性警告
· 从依赖 Mod 页面退出时会保留原本卡片的展开状态（#2712，@WorldHim）
· 更新迁移 Mojang 账户的说明（#2718，@Louis-ZYH，by @JingHai-Lingyun）
· 新主页预设：由 @MFn233 制作的简单主页
· 更新 9 月赞助者列表，感谢大家的支持！
· 修复了以下 11 个 Bug
　　不应显示如何禁用 Mod 的提示（#2686，@Sunday40847）
　　向 PCL 拖入压缩包中的文件可能导致出错（#2690，@PPicku）
　　自定义主页教学文件无法加载（#2695，@ItIsNotAPlayer）
　　回声洞会出现自定义“你知道吗”的内容（#2704，@kong-que-dnf123）
　　安装后的 json 有无效的 minecraftArguments（#2705，@zkitefly）
　　分析特定崩溃日志时出错（#2706，@pptawa）
　　无法安装 MCBBS 整合包中的预览版 OptiFine（#2730，@EMUMUI）
　　个别 Mod 的信息读取有误（#2732，@zkitefly）
　　个别情况下下载自定义文件结束后合并时出错（#2742，@Bandysol）
　　无法自动搜索到 Zulu Java（#2752，@winver1234）
　　使用多个 {hint}、{cave} 时替换的内容全部相同

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.9'>2.6.9</a>.</em>

**Comment by whwdzg** [comment]
好耶
